src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -89,6 +90,14 @@ internal override CSharpSyntaxNode Root
             }
         }
 
+        internal bool ContainsNullCheckedParameter
```

Comment:
>ContainsNullCheckedParameter [](start = 22, length = 28)

Is this used by `CSharpOperationFactory` only? If so, it seems like the check can be moved there.